src/components/molecules/PageNation/index.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="pageNation">
+    <ul class="pageNation__items">
+      <li>
+        <router-link
+          :disabled="pageNum === 1"
+          white
+          large
+          bg-theme-color
+          hover-opacity
+          :to="`articles?page=1`"
+        >
+          1
+        </router-link>
+      </li>
+      <li>
+        <app-text
+          class="pageNation__items__text"
+          bold
+        >
+          ...
+        </app-text>
+      </li>
+      <li
+        v-for="page in totalPage"
```

Comment:
[imo]

`totalPage` だとページ総数のようなイメージを受けるので、`middlePages` といったように記述できるとぱっと見わかりやすいかなと思いました。
基本的に配列を表す変数名は複数形にした方がわかりやすいですし、慣習的に使用されています。


```suggestion
        v-for="page in middlePages"
```

Reply:
前回指摘された箇所の修正しました。
1fbf08ff1cc1bd85d33f8e1ccdc07cdad7a1c4e2